repos/system_upgrade/el7toel8/actors/targetuserspacecreator/libraries/userspacegen.py

```diff
@@ -0,0 +1,127 @@
+import itertools
+import os
+
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.actor import constants
+from leapp.libraries.common import dnfplugin, mounting, overlaygen, rhsm, utils
+from leapp.libraries.stdlib import api, run
+from leapp.models import (OSReleaseFacts, RequiredTargetUserspacePackages,
+                          SourceRHSMInfo, TargetUserSpaceInfo)
+
+
+def _logger():
```

Comment:
Same question why do we need a function here

Reply:
Same answer